release/charts/arguments.go

```diff
@@ -0,0 +1,132 @@
+package charts
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"sort"
+
+	"github.com/rancher/ecm-distro-tools/cmd/release/config"
+)
+
+type status struct {
+	ToRelease     map[string][]asset `json:"to_be_released"`
+	ToForwardPort map[string][]asset `json:"to_be_forward_ported"`
+}
+
+type asset struct {
+	Version string `json:"version"`
+}
+
+// loadState will load the lifecycle-status state from an existing state.json file at charts repo
+func loadState(filePath string) (*status, error) {
+	s := &status{}
+
+	file, err := os.Open(filePath)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := json.NewDecoder(file).Decode(&s); err != nil {
+		return nil, err
+	}
+
+	return s, nil
+}
+
+// BranchArgs will return the list of available branch version lines
+func BranchArgs() []string {
+	return []string{"2.9", "2.8"}
```

Comment:
Can this be part of configuration to avoid code changes when things change?

Reply:
Implementing